sources/packages/forms/src/form-definitions/sfaa2025-26-ft.json

```diff
@@ -2528,9 +2528,218 @@
+                  "storage": "url",
+                  "dir": "Estranged Details",
+                  "webcam": false,
+                  "fileTypes": [
+                    {
+                      "label": "",
+                      "value": ""
+                    }
+                  ],
+                  "filePattern": ".pdf,.doc,.docx,.jpg,.png,.txt",
+                  "fileMaxSize": "15MB",
+                  "multiple": true,
+                  "validate": {
+                    "required": true,
+                    "customMessage": "You must upload at least one file, and the total number of uploaded files cannot exceed {{data.maxUploadedFiles}}.",
+                    "custom": "const files = data[instance.key] || []; valid = files.length > data.maxUploadedFiles ? false : true;"
+                  },
+                  "key": "estrangedFileUpload",
+                  "type": "file",
+                  "url": "student/files",
+                  "input": true
+                },
+                {
+                  "label": "Columns",
+                  "columns": [
+                    {
+                      "components": [
+                        {
+                          "html": "<p>We accept <strong>JPG, PNG, DOC, DOCX, PDF, TXT</strong></p>\n",
+                          "label": "Content",
+                          "refreshOnChange": false,
+                          "key": "content52",
+                          "type": "content",
+                          "input": false,
+                          "tableView": false,
+                          "hideOnChildrenHidden": false
+                        }
+                      ],
+                      "width": 6,
+                      "offset": 0,
+                      "push": 0,
+                      "pull": 0,
+                      "size": "md",
+                      "currentWidth": 6
+                    },
+                    {
+                      "components": [
+                        {
+                          "html": "<p style=\"text-align:right;\">15MB file limit each</p>\n",
+                          "label": "Content",
+                          "refreshOnChange": false,
+                          "key": "content53",
+                          "type": "content",
+                          "input": false,
+                          "tableView": false,
+                          "hideOnChildrenHidden": false
+                        }
+                      ],
+                      "width": 6,
+                      "offset": 0,
+                      "push": 0,
+                      "pull": 0,
+                      "size": "md",
+                      "currentWidth": 6
+                    }
+                  ],
+                  "hideOnChildrenHidden": false,
+                  "key": "columns10",
+                  "type": "columns",
+                  "input": false,
+                  "tableView": false
+                }
+              ]
```

Comment:
There is a simplified component that can be copied from below.
![image](https://github.com/user-attachments/assets/5e589850-669a-49c6-a271-bb5c42c88d39)

```json
{
  "label": "Content",
  "input": false,
  "className": "file-upload-info-container",
  "content": "<p>We accept <strong>JPG, PNG, DOC, DOCX, PDF, TXT</strong></p>\n<p>15MB file limit each</p>",
  "type": "htmlelement",
  "hideLabel": true,
  "tableView": false
}
```

Reply:
updated.

Reply:
Please check if the changes were pushed.

Reply:
Changes were sitting waiting to be synced. Updated now.